gitlab.go

```diff
@@ -134,6 +134,7 @@ type Client struct {
 	Settings        *SettingsService
 	SystemHooks     *SystemHooksService
 	Users           *UsersService
+	Builds          *BuildsService
```

Comment:
_nitting_ Could you please insert the new `Builds` service following alphabetic ordering?